### ISSUE TITLE
Timezone America/Sao_Paulo is not consistent

### ISSUE THREAD

**Issue by mfandre** [huzaifahj/slot-calculator#5]
This example wont work, it should return available true. :(

```
import { getSlots } from "slot-calculator";
import { DateTime, Settings } from "luxon";
//Settings.defaultZone = "UTC";
const dateTimeRef = DateTime.utc(2023, 1, 24);
const weekday = dateTimeRef.weekdayLong;
console.log(weekday)

const allSlots = getSlots({
  from: "2023-01-24T12:00:00-03:00",
  to: "2023-01-24T13:00:00-03:00", 
  outputTimezone: "America/Sao_Paulo",
  duration: 60,
  unavailability:[],
  availability: [
    {
        day: "segunda",
        from: "09:00",
        to: "19:00",
        timezone: "America/Sao_Paulo"
    },{
        day: "terca-feira",
        from: "01:00",
        to: "23:00",
        timezone: "America/Sao_Paulo"
    },
    {
        day: "quarta-feira",
        from: "09:00",
        to: "19:00",
        timezone: "America/Sao_Paulo"
    },
    {
        day: "quinta-feira",
        from: "09:00",
        to: "19:00",
        timezone: "America/Sao_Paulo"
    },
    {
        day: "sexta-feira",
        from: "09:00",
        to: "19:00",
        timezone: "America/Sao_Paulo"
    }
  ]
});

console.log(allSlots);

```

**Comment by huzaifahj** [comment]
Thanks for creating this issue @mfandre 

I believe this is because you are entering weekdays like `segunda` instead of `Monday`.

Internationalisation could definitely be better supported for weekdays, I will look into this for you

**Comment by huzaifahj** [comment]
@mfandre Fixed in latest release 7d0b064 

You can now do something like this:

```ts
getSlots({
  from: dateTimeRef.plus({ hour: 3 }).toISO(),
  to: dateTimeRef.plus({ hour: 5 }).toISO(),
  outputTimezone: "America/Sao_Paulo",
  duration: 60,
  availability: [
    {
      day: {
        text: "sábado",
        locale: "pt-br",
      },
      from: "00:00",
      to: "01:00",
      timezone: "America/Sao_Paulo",
    },
  ],
})
```
